code/modules/arcade/claw_game.dm

```diff
@@ -0,0 +1,84 @@
+/var/claw_game_html = null
+
+/obj/machinery/arcade/claw
+	name = "Claw Game"
+	desc = "One of the most infuriating ways to win a toy."
+	icon = 'icons/obj/arcade.dmi'
+	icon_state = "clawmachine_1_on"
+	token_price = 15
+	window_name = "Claw Game"
+	var/machine_image = "_1"
+	var/bonus_prize_chance = 5		//chance to dispense a SECOND prize if you win, increased by matter bin rating
+
+	//This is to make sure the images are available
+	var/list/img_resources = list('icons/obj/arcade_images/backgroundsprite.png',
+								'icons/obj/arcade_images/clawpieces.png',
+								'icons/obj/arcade_images/crane_bot.png',
+								'icons/obj/arcade_images/crane_top.png',
+								'icons/obj/arcade_images/prize_inside.png',
+								'icons/obj/arcade_images/prizeorbs.png')
```

Comment:
you've used the wrong typepath for all of these that you actually added them as..